backend/src/routes/applications.js

```diff
@@ -95,6 +96,14 @@ router.get(
   },
 )
 
+/**
+ * Get Funding Agreement PDF by ID
+ */
+router.get('/:applicationId/pdf', passport.authenticate('jwt', { session: false }), isValidBackendToken, validatePermission(PERMISSIONS.VIEW_APPLICATIONS), (req, res) => {
```

Comment:
should we add this param validation? "[param('applicationId', 'URL param: [applicationId] is required').not().isEmpty()],"

Reply:
yup!!